src/ai/ai.cpp

```diff
@@ -265,50 +458,241 @@ static bool ai_has_mutual_enemy(sys::state& state, dcon::nation_id from, dcon::n
 	return false;
 }
 
-constexpr inline float ally_overestimate = 2.f;
-
 bool ai_will_accept_alliance(sys::state& state, dcon::nation_id target, dcon::nation_id from) {
-	if(!state.world.nation_get_ai_is_threatened(target))
```

Comment:
why have you decided to remove this check? If the AI is not threatened, it means that it does not think that it needs more allies to deter an attack. At that point, additional allies provide no material benefit while limiting its expansion options.